bin/cloudgov/domains-for-redirects/domains.json

```diff
@@ -6,8 +6,6 @@
         "apps.usa.gov",
         "benefits-tool.usa.gov",
         "blog.gobiernousa.gov",
-        "business.usa.gov",
```

Comment:
Phooey. I forgot about this one. `business.usa.gov` has not been decommissioned and I don't know why not, but we can't remove this right now. 

Let's hold off on this; it uses the same block of redirects.